migrations/sql/V2020.04.17.12.48__add_context_props_to_document_template.sql

```diff
@@ -0,0 +1,132 @@
+UPDATE document_template SET form_spec_json = '[
+    {
+      "id": "letter_dt",
+      "label": "Letter Date",
+      "type": "DATE",
+      "placeholder": null,
+      "required": true
+    },
+    {
+      "id": "mine_no",
+      "label": "Mine Number",
+      "type": "FIELD",
+      "placeholder": "Enter the mine number",
+      "required": true,
+      "relative-data-path": "mine.mine_no",
+      "read-only": true
+    },
+    {
+      "id": "proponent_address",
+      "label": "Proponent Address",
+      "type": "FIELD",
+      "placeholder": "Enter the proponent''s address",
+      "required": true
+    },
+    {
+      "id": "proponent_name",
+      "label": "Proponent Name",
+      "type": "FIELD",
+      "placeholder": "Enter the proponent''s name",
+      "required": false
+    },
+    {
+      "id": "emailed_to",
+      "label": "Emailed to",
+      "type": "FIELD",
+      "placeholder": "Enter the name of the email recipient",
+      "required": false
+    },
+    {
+      "id": "property",
+      "label": "Property",
+      "type": "FIELD",
+      "placeholder": "Enter the property",
+      "required": true,
+      "relative-data-path": "now_application.property_name",
+      "read-only": true
+    },
+    {
+      "id": "application_dt",
+      "label": "Application Date",
+      "type": "DATE",
+      "placeholder": null,
+      "required": true,
+      "relative-data-path": "now_application.submitted_date"
+    },
+    {
+      "id": "exploration_type",
+      "label": "Exploration Type",
+      "type": "FIELD",
+      "placeholder": "Enter the exploration type",
+      "required": true
+    },
+    {
+      "id": "bond_inc_amt",
+      "label": "Bond Amount",
+      "type": "FIELD",
+      "placeholder": "Enter the bond amount",
+      "required": true,
+      "context-props": ["currency"]
```

Comment:
This is the only change.

Reply:
I updated Seed data, there are update statements that will run to set these values, this can just be changed there. my guess is that would actually overwrite this given it runs after.

Reply:
True, I was skeptical about the need for this file. I can delete it then?